app/components/common/highLightedContent/index.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from "react";
 import { Link } from "react-router-dom";
-import Icon from "../../../icons";
+import { withStyles } from "../../../helpers/withStylesHelper";
+const styles = require("./highLightedContent.scss");
 
 interface SearchQueryContentProps {
```

Comment:
Please change this interface name and component className following the filename. (HighLightContentProps, HighLightContent)